client/layout/sidebar/style.scss

```diff
@@ -277,3 +277,17 @@
 .sidebar__menu-wrapper {
 	border-top: 1px solid var( --color-sidebar-border );
 }
+
+.sidebar__menu-link-badge {
+	display: inline-block;
+	padding: 1px 6px;
+	border: solid 1px var( --color-border );
+	border-radius: 12px;
+	font-size: $font-body-extra-small;
+	line-height: 14px;
+	text-align: center;
+	color: var( --color-text-inverted );
+	border-color: var( --color-accent );
```

Comment:
This property overrides the color on the `border` few lines earlier. One of them is redundant.

Reply:
Right, Thanks!